examples/hosts/app-integration/external-data/src/mock-customer-service/service.ts

```diff
@@ -10,7 +10,29 @@ import express from "express";
 import fetch from "node-fetch";
 
 import { assertValidTaskData, TaskData } from "../model-interface";
+import { ClientManager, ExternalTaskListId } from "../utilities";
 
+/**
+ * Registers for webhook on receiving a specific resource to register for.
+ */
+async function registerForWebhook(
```

Comment:
I think we can just inline the contents of this function in its only call site and document what it's accomplishing there.

Reply:
Hmm is this how you mean? I'm replacing it with `Register with external data service for webhook notifications.`

Reply:
I mean that I don't see much point in this code being extracted into a function since we only call it in one place, and it's basically a single (not visually, functionally). We could just do `await fetch(externalDataServiceWebhookRegistrationUrl, ... ` where we currently call `registerForWebhook()`.

Reply:
I didn't quite understand what you meant the first time. Got it. You mean move the whole function down into the call site. Yup, makes sense.